### PR TITLE
Add a tool to print tsfileResources (each device's start and end time)

### PR DESCRIPTION
**Pull request by jixuan1989**
The tool can help users knowing some info about tsfiles, and may help users to analyze the query time cost..

### CODE REVIEW

File: server/src/assembly/resources/tools/print-tsfile-resource-files.bat [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+goto :eof
+
+REM -----------------------------------------------------------------------------
+:okClasspath
+
+rem echo CLASSPATH: %CLASSPATH%
+set IOTDB_DATA=%IOTDB_HOME%\data
+set IOTDB_WAL=%IOTDB_DATA%\wal
+
+IF EXIST "%IOTDB_WAL%" (
```
Comment (qiaojialin) [inline server/src/assembly/resources/tools/print-tsfile-resource-files.bat]: why check IOTDB_WAL?

File: server/src/assembly/resources/tools/print-tsfile-resource-files.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+fi
+
+IOTDB_CONF=${IOTDB_HOME}/conf
+# IOTDB_LOGS=${IOTDB_HOME}/logs
+
+#if [ -f "$IOTDB_CONF/iotdb-env.sh" ]; then
+#    . "$IOTDB_CONF/iotdb-env.sh"
+#else
+#    echo "can't find $IOTDB_CONF/iotdb-env.sh"
+#fi
```
Comment (qiaojialin) [inline server/src/assembly/resources/tools/print-tsfile-resource-files.sh]: remove this?

File: server/src/assembly/resources/tools/print-tsfile-resource-files.bat (hunk 1 above)
Comment (jixuan1989) [reply server/src/assembly/resources/tools/print-tsfile-resource-files.bat]: ah... because I copy it from checkWAL.sh...

Comment (jixuan1989) [reply server/src/assembly/resources/tools/print-tsfile-resource-files.bat]: fixed

File: server/src/assembly/resources/tools/print-tsfile-resource-files.sh (hunk 2 above)
Comment (jixuan1989) [reply server/src/assembly/resources/tools/print-tsfile-resource-files.sh]: fixed

**Pull request merged**